Content.Server/Drone/DroneSystem.cs

```diff
@@ -69,9 +70,16 @@ private void OnMobStateChanged(EntityUid uid, DroneComponent drone, MobStateChan
             {
                 var body = Comp<SharedBodyComponent>(uid); //There's no way something can have a mobstate but not a body...
 
-                foreach (var item in drone.ToolUids)
+                foreach (var item in drone.ToolUids.Select((value, i) => ( value, i )))
                 {
-                    EntityManager.DeleteEntity(item);
+                    if (drone.Tools[item.i].PrototypeId == "ClothingBackpackSatchelDrone")
+                    {
+                        RemComp<UnremoveableComponent>(item.value);
+                    }
```

Comment:
absolutely do not hardcode this. make a tag or component

Reply:
got it, i'll change this to a tag

Reply:
complete, checks for tag instead